backend/module/chaos/serverexperimentation/xds/xds_cache.go

```diff
@@ -79,7 +80,10 @@ func refreshCache(ctx context.Context, storer experimentstore.Storer, snapshotCa
 	}
 
 	// Create/Update experiments
+	activeFaults := 0
```

Comment:
i think this is equal to `len(clusterFaultMap)` instead of maintaining a separate counter